environment-scheme-sim.yml

```diff
@@ -0,0 +1,8 @@
+name: scm_scheme_sim
```

Comment:
Just noting that the existence of this yaml isn't documented yet

Reply:
Documentation on yal script added